ViewModel/AggregatedStageDamage/FrequencyElementWrapper.cs

```diff
@@ -0,0 +1,39 @@
+﻿using HEC.FDA.ViewModel.FrequencyRelationships;
+
+namespace HEC.FDA.ViewModel.AggregatedStageDamage
+{
+    /// <summary>
+    /// This class exists for the sole purpose of having a blank row in the combobox
```

Comment:
I'd rather not just hand jam something weird in. Is there a better way to do this?